services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
```

Comment:
The return type for `create_movie_session` should be `MovieSession` instead of `QuerySet` since it returns a single instance of `MovieSession`.